docs/source/quickstart.rst

```diff
@@ -193,24 +193,50 @@ In the examples above, MLflow logs data to the local filesystem of the machine i
 To manage results centrally or share them across a team, you can configure MLflow to log to a remote
 tracking server. To get access to a remote tracking server:
 
-- :ref:`Launch a tracking server <tracking_server>` on a remote machine.
+Launch a tracking server on a remote machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:ref:`Launch a tracking server <tracking_server>` on a remote machine.
 
-- Alternatively, sign up for `Databricks Community Edition <https://databricks.com/try-databricks>`_,
-  which comes bundled with a hosted tracking server. Note that
-  Community Edition is intended for quick experimentation rather than production use cases.
-  After signing up, run ``databricks configure`` to create a credentials file for MLflow, specifying
-  https://community.cloud.databricks.com as the host.
+You can then :ref:`log to the remote tracking server <logging_to_a_tracking_server>`. If
+running your own tracking server, add the following to the start of your program, updating
+``tracking_uri`` to your server's URI:
 
-You can then :ref:`log to the remote tracking server <logging_to_a_tracking_server>`, e.g. by
-adding the following to the start of your program:
+  .. code-section::
+
+    .. code-block:: python
+
+        import mlflow
+        # Update this to your tracking server's URI
+        tracking_uri = "..."
```

Comment:
You can remove this line and just show the URI inline on the next one. For example `mlflow.set_tracking_uri("http://YOUR_SERVER:4040")`.